src/components/BulkEnrollmentPage/stepper/AddLearnersStep.test.jsx

```diff
@@ -82,8 +82,6 @@ describe('AddLearnersStep', () => {
     expect(await screen.findByText('Search Email')).toBeInTheDocument();
     await act(() => mockTableData);
     userEvent.type(screen.getByPlaceholderText('Search Email'), 'beAR');
-    expect(await screen.findByText(/Filtered by userEmail/)).toBeInTheDocument();
```

Comment:
you can actually enhance this assertoin by checking specifically that the field does not exist in the dom, sth like 

```
expect(await screen.queryByText(/Filtered by userEmail/)).not.toBeInTheDocument();
```

Check the docs for React testing library I think you need to use queryBy instead of findBy for testing absence ... not 100%

Reply:
https://testing-library.com/docs/guide-disappearance/#asserting-elements-are-not-present